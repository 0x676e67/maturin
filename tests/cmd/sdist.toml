bin.name = "maturin"
args = "sdist --help"
