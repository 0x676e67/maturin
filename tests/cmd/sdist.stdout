maturin[EXE]-sdist 
Build only a source distribution (sdist) without compiling.

Building a source distribution requires a pyproject.toml with a `[build-system]` table.

This command is a workaround for [pypa/pip#6041](https://github.com/pypa/pip/issues/6041)

USAGE:
    maturin[EXE] sdist [OPTIONS]

OPTIONS:
    -m, --manifest-path <MANIFEST_PATH>
            The path to the Cargo.toml

    -o, --out <OUT>
            The directory to store the built wheels in. Defaults to a new "wheels" directory in the
            project's target directory

    -h, --help
            Print help information
